instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import CommentSection from '../CommentSection/CommentSection.js';
+
+const PostContainer = props => {
+    return (
+        <div>
+            {props.postData.map(post => (
+                <div>
+                    <div>
+                        <img src={post.thumbnailUrl} alt="user thumbnail"/>                    
+                        <h2>{post.username}</h2>
```

Comment:
You also need a key here since you are mapping. It would also serve you to separate alot of this JSX into multiple sub-components this way you can use the post container to just Orchestrate them all.

```js
 return (
        <div>
            {props.postData.map(post => ( // Even this map can be moved to the parent and pass down 1 post obj.
                <div>
                    <PostHeader OrginalPoster={props.stuff} />
                    <PostContent imgSRC={post.bla bla} />
                    <CommentSection comments={post.comments} />
                </div>
            ))}
        </div>    
    )
```